sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -312,13 +330,29 @@ export class BatchingReceiver extends MessageReceiver {
         }
       };
 
+      const cleanupBeforeReject = (
+        receiver: Receiver | undefined,
+        onReceiveErrorHandlerToRemove: OnAmqpEvent
```

Comment:
Why do you pass in `onReceiveErrorHandlerToRemove` but not the other handlers?

Reply:
This was me eliminating an eslint warning about using a variable before it was defined. I could pass the other ones in for consistency but in reality I'd rather not pass _any_ of them in.

I really didn't want to (in this particular PR) restructure the Promise anymore than I had to, even though it _really_ needs it.

Reply:
I'll toss a comment on there. That entire Promise is basically one big TODO at this point.

Reply:
Fixed.